deploy-edpm-reuse.yaml

```diff
@@ -2,6 +2,22 @@
 - name: Manage unique ID
   ansible.builtin.import_playbook: playbooks/unique-id.yml
 
+- name: Reboot controller-0 to make sure there are no running deployments
+  hosts: controller-0
+  gather_facts: false
+  tasks:
+    - name: Reboot controller-0
+      ansible.builtin.reboot:
+        reboot_timeout: 600
+      register: reboot_result
+      become: true
+
+    - name: Wait for controller-0 to come back online
+      ansible.builtin.wait_for_connection:
+        delay: 10
+        timeout: 600
+      when: reboot_result is changed
```

Comment:
This line is triggering `ansible-lint` error. I'm not sure about a better way to implement that. We might consider disabling the linter on it.

Reply:
Scratch above. I think I suggested slightly better solution.